deployments/deploy-ssh.sh

```diff
@@ -649,6 +649,8 @@ spec:
       value: "${TARGET_PXBACKUP_VERSION}"
     - name: TARGET_STORK_VERSION
       value: "${TARGET_STORK_VERSION}"
+    - name: PX_BACKUP_HELM_REPO_BRANCH
```

Comment:
can we change this to TARGET_PX_BACKUP_HELM_REPO_BRANCH in align with other variables.

Reply:
We are not migrating from one branch to another which is why I thought `TARGET` will not make sense. If we are upgrading from one version to another it would make sense.